src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -662,19 +662,37 @@ public function setAutowiringTypes(array $types)
      */
     public function isAutowired()
     {
+        return !empty($this->autowired);
+    }
+
+    /**
+     * Gets autowired methods.
+     *
+     * @return string[]
+     */
+    public function getAutowired()
+    {
         return $this->autowired;
     }
 
     /**
      * Sets autowired.
      *
-     * @param bool $autowired
+     * Allowed values:
+     *   - true: constructor autowiring
+     *   - array('__construct', 'set*', 'setBar'): autowire whitelisted methods only
```

Comment:
`setBar` should already be matched because `set*`. What about using `initialize`  instead of `setBar` to make this method easier to understand or use more specific patterns such as `['setApp*', 'setBar']`?